std/http/file_server.ts

```diff
@@ -103,7 +104,10 @@ async function serveFile(
   const [file, fileInfo] = await Promise.all([open(filePath), stat(filePath)]);
   const headers = new Headers();
   headers.set("content-length", fileInfo.size.toString());
-  headers.set("content-type", "text/plain; charset=utf-8");
```

Comment:
In fact, this line of code was added by me. You know, we need to browse the `typescript` file frequently. Unfortunately, `.ts` is the extension of the video by default.

Reply:
You can see https://github.com/denoland/deno/issues/4056, there are some related discussions

Reply:
We can make exception for `.ts` and `.js` as a workaround.
* `.ts` → `text/plain`
* `.js` → `text/javascript`

Reply:
waiting for @ry 's suggestion

Reply:
@zhmushan Yes I remember when we landed that - but I've changed my mind now, I think it's more useful for file_server to act as a real file server.

Reply:
We should change the media_types so that .ts properly maps to typescript.